rosidl_runtime_rs/src/traits.rs

```diff
@@ -42,7 +42,7 @@ pub trait RmwMessage: Clone + Debug + Default + Send + Sync + Message {
     const TYPE_NAME: &'static str;
 
     /// Get a pointer to the correct `rosidl_message_type_support_t` structure.
-    fn get_type_support() -> libc::uintptr_t;
+    fn get_type_support() -> usize;
```

Comment:
While we're here – could we make this pointer and the one in the `Service` trait a `*const c_void`?